fem/datacollection.cpp

```diff
@@ -164,8 +164,8 @@ void DataCollection::SetFormat(int fmt)
 void DataCollection::SetCompression(bool comp)
 {
    compression = comp;
-#ifndef MFEM_USE_GZSTREAM
-   MFEM_VERIFY(!compression, "GZStream not enabled in MFEM build.");
+#ifdef MFEM_USE_GZSTREAM
+   MFEM_ASSERT(!compression, "GZStream not enabled in MFEM build.");
```

Comment:
This was a bugfix I added -- I don't think it is correct to revert it.

Reply:
Ok, I'll re-revert it

Reply:
`done`